module/zfs/Makefile.in

```diff
@@ -114,6 +115,7 @@ $(MODULE)-objs += zrlock.o
 $(MODULE)-objs += zvol.o
 $(MODULE)-objs += dsl_destroy.o
 $(MODULE)-objs += dsl_userhold.o
+@CONFIG_QAT_TRUE@$(MODULE)-objs += qat_compress.o
```

Comment:
We don't need to make this conditional.  We can always build qat_compress.c as long as the contents are wrapper with HAVE_QAT.